paramak/parametric_components/toroidal_field_coil_princeton_d.py

```diff
@@ -60,6 +62,7 @@ def __init__(
         self.number_of_coils = number_of_coils
         self.vertical_displacement = vertical_displacement
         self.with_inner_leg = with_inner_leg
+        self.rotation_angle = rotation_angle
```

Comment:
This is maybe not needed since 360.0 is already the default value at RotateMixedShape level